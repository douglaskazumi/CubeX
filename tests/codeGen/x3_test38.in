"Modern Compiler Implementation in Java,"
"2nd ed."
"Andrew Appel and Jens Palsberg,"
"Cambridge University Press,"
"2002."
"ISBN 0-521-82060-X."
"Compilersâ€”Principles,"
"Techniques and Tools"
"(The Dragon Book),"
"2nd ed."
"Alfred Aho,"
"Monica Lam,"
"Ravi Sethi and Jeffrey D."
"Ullman."
"Addison-Wesley,"
"2006."
"ISBN:"
"0-321-48681-1"
"Engineering a Compiler."
"Keith Cooper and Linda Torczon."
"Elsevier/Morgan Kaufmann,"
"2004."
"ISBN 1-55860-698-8."
"Advanced Compiler Design and Implementation."
"Steve Muchnick."
"Morgan Kaufmann Publishers,"
"1997."
"ISBN 1-558-60320-4."
"Principles of Program Analysis."
"Flemming Nielson,"
"Hanne Riis Nielson,"
"and Chris Hankin."
"Springer,"
"2004."
"ISBN 3540654100."