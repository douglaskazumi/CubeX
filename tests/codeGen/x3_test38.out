helloWAT
fooWAT
Modern Compiler Implementation in Java,WAT
2nd ed.WAT
Andrew Appel and Jens Palsberg,WAT
Cambridge University Press,WAT
2002.WAT
ISBN 0-521-82060-X.WAT
Compilersâ€”Principles,WAT
Techniques and ToolsWAT
(The Dragon Book),WAT
2nd ed.WAT
Alfred Aho,WAT
Monica Lam,WAT
Ravi Sethi and Jeffrey D.WAT
Ullman.WAT
Addison-Wesley,WAT
2006.WAT
ISBN:WAT
0-321-48681-1WAT
Engineering a Compiler.WAT
Keith Cooper and Linda Torczon.WAT
Elsevier/Morgan Kaufmann,WAT
2004.WAT
ISBN 1-55860-698-8.WAT
Advanced Compiler Design and Implementation.WAT
Steve Muchnick.WAT
Morgan Kaufmann Publishers,WAT
1997.WAT
ISBN 1-558-60320-4.WAT
Principles of Program Analysis.WAT
Flemming Nielson,WAT
Hanne Riis Nielson,WAT
and Chris Hankin.WAT
Springer,WAT
2004.WAT
ISBN 3540654100.WAT
